docs/source/daml/reference/choices.rst

```diff
@@ -91,9 +140,8 @@ Non-consuming choices
 
 - ``nonconsuming`` keyword. Optional.
 - Makes a choice non-consuming: that is, exercising the choice does not archive the contract.
-
-  By default, choices are *consuming*: when a choice on a contract is exercised, that contract instance is *archived*. Archived means that it's permanently marked as being inactive, and no more choices can be exercised on it, though it still exists on the ledger.
-- This is useful in the many situations when you want to be able to exercise a choice more than once.
+- Contracts created in the choice body are known only to the signatories and controllers of the contracts and not made known to the observers of the contract on which the choice was exercised.
```

Comment:
And here

Reply:
Solved in https://github.com/digital-asset/daml/pull/4134/commits/11f52f472e3cb698771d149a1e0d4ac0a464f02a